generated/thrift/rpc.thrift

```diff
@@ -49,11 +49,17 @@ service Node {
 	FetchRawBatchResult fetchRawBatch(1: FetchRawBatchRequest req) throws (1: Error err)
 	FetchBlocksResult fetchBlocks(1: FetchBlocksRequest req) throws (1: Error err)
 	FetchBlocksMetadataResult fetchBlocksMetadata(1: FetchBlocksMetadataRequest req) throws (1: Error err)
+	void truncateNamespace(1: TruncateNamespaceRequest req) throws (1: Error err)
+}
+
+struct TruncateNamespaceRequest {
+    1: required string ns
```

Comment:
Any reason to use `ns` instead of more descriptive `namespace` here?  This will also be our HTTP/JSON API too so might be best to make it `namespace` here and in the other structs.